api/rest/device.def.js

```diff
@@ -49,6 +50,21 @@ DeviceActivateOption.prototype = new ConnectionOptions();
 DeviceActivateOption.prototype.constructor = DeviceActivateOption;
 IoTKiT.DeviceActivateOption = DeviceActivateOption;
 
+
+
+function DeviceGetMetadataOption (data) {
+    this.pathname = common.buildPath(apiconf.path.device.get, data.deviceId);

```

Comment:
If i understand it, we're just getting device, it's simple GET /device endpoint so name is a little bit confusing, it just DeviceGetOption . Current name sugest that we have got already device and now requesting for something additional